js/client-chat.js

```diff
@@ -1266,13 +1266,15 @@
 					break;
 
 				case 'uhtml':
-					this.$chat.append('<div class="notice uhtml-' + toId(row[1]) + '">' + Tools.sanitizeHTML(row.slice(2).join('|')) + '</div>');
-					break;
-
 				case 'uhtmlchange':
 					var $elements = this.$chat.find('div.uhtml-' + toId(row[1]));
-					if (!$elements.length) break;
-					$elements.html(Tools.sanitizeHTML(row.slice(2).join('|')));
+					if (!toId(row.slice(2).join('|'))) {
```

Comment:
You should probably cache `row.slice(2).join('|')` somewhere. Instead of doing it three times, one in each branch.

Reply:
good catch